lang/java/avro/src/main/java/org/apache/avro/package.html

```diff
@@ -26,8 +26,8 @@
   org.apache.avro.io.DatumReader} and {@link
   org.apache.avro.io.DatumWriter} implementations.  Generic
   implementations are provided in the {@link org.apache.avro.generic}
-  package.  A {@link org.apache.avro.compiler.specific.SpecificCompiler
-  compiler} can generate specific java classes and interfaces for
+  package.  A {@link <a href="javascript:">org.apache.avro.compiler.specific.SpecificCompiler
```

Comment:
I'm not sure this is right -- oddly enough, the original actually works in the official site doc: https://avro.apache.org/docs/1.9.1/api/java/org/apache/avro/package-summary.html#package.description

When I built the site myself, the link is now inoperative...

If the warning is because of SpecificCompiler is in a different package, but we want to link to it anyway, perhaps a relative HTML link would be the best solution instead of a `{@link}`?

```<a href="./compiler/specific/SpecificCompiler.html">SpecificCompiler</a>```

Reply:
Thank you,it has been updated